Utility/Utility.Public/Include/Utility/JGExceptions.hpp

```diff
@@ -40,7 +40,7 @@ class JuniorgramException : public std::exception
 private:
     std::string   _msg;
     std::string   _file;
-    std::uint16_t _line;
+    [[maybe_unused]] std::uint16_t _line;
```

Comment:
What is the premise of [[maybe_unused]] attribute?

Reply:
goole it